x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/onboarding_steps.test.tsx

```diff
@@ -24,7 +24,6 @@ const account = {
   isAdmin: true,
   canCreatePersonalSources: true,
   groups: [],
-  supportEligible: true,
```

Comment:
@scottybollinger Another heads up that I also deleted this, it doesn't seem to be in the initial AppLogic API spec and I don't see it used anywhere in ent-search either